frontend/src/components/pages/search/Search.tsx

```diff
@@ -66,7 +67,7 @@ const Search: React.FC = () => {
       isOnline: checkboxFilters.isOnline
     };
     e.preventDefault();
-    console.log(JSON.stringify(request));
+    getUsersByParams(`http://127.0.0.1:8000/api/users`, JSON.stringify(requestBody)).then(users => setUsers(users));
```

Comment:
move the url and json stringify things into service (so the function just get params and returns data, everything else is hidden inside realisation (Encapsulation from OOD lecture))